ProcessLib/NumericsConfig.h

```diff
@@ -129,4 +129,7 @@ static_assert(std::is_same<detail::GlobalMatrixType::IndexType,
 /// GlobalMatrixType::IndexType and the GlobalVectorType::IndexType.
 using GlobalIndexType = detail::GlobalMatrixType::IndexType;
 
+/// A type for scalar number
+using ScalarType = detail::GlobalMatrixType::ScalarType;
```

Comment:
But there is a copy anyway!

Regarding some special types: vtk (as of version 6.3) supports only 32 and 64 bit floating point number types. No complex, no long double. Please correct me if I'm wrong.

I'm not very knowledgeable about existing problems/benchmarks where complex type is used. Would you please point to such examples (in ogs5 for example)?